code/modules/client/preference_setup/general/03_body_ch.dm

```diff
@@ -0,0 +1,28 @@
+// Chomp additions to character load/save logic
+// /datum/category_item/player_setup_item/general/body/content cannot be overridden here easily as the proc creates a list of html data
+// perhaps one could get that list here, and sort through sections to rewrite it's content...? that seems inefficent having to loop through the list.
+
+
+/datum/category_item/player_setup_item/general/body/copy_to_mob(var/mob/living/carbon/human/character)
+	. = ..()
+	if(character.species.digi_allowed)
+		character.digitigrade = pref.digitigrade
```

Comment:
I feel like this spot could use a sanity check to make sure the pref has this option saved, otherwise it slaps that toggle var into a null due to old savefile lacking the value. Also there seems to be a lot of defined 0 vars in this PR that should probably use FALSE instead for consistency.

Reply:
I wanted to keep digitigrade itself as a value so i could allow for future different leg styles but there may be a better way to do this (?) - i will however add the sanity check and check elsewhere for this